docs/reference-manual/common-practices/looping.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: default
+title: "Looping"
+
+parent: Common practices
+grand_parent: Reference manual
+has_children: false
+
+nav_order: 1
+
+permalink: /docs/reference-manual/common-practices/looping.html
+category: "site-score"
+
+description: "Looping"
+
+tag: "Presentation"
+visible: true
+---
+
+# Looping
+
+## Process Loops
+
+Time based processes like automations, media and sub-scenarios can be looped for as long as their parent interval is active. When selecting a process, eiher form the object window or by clicking on the it's upper margin, the loop option will appear in the `Inspector`.
+
+![Process looping](/score-docs/assets/images/reference-manual/common-practices/loopProcess.gif "Process looping")
+
+## Loop with transitions
+
+To enable the repetition of certain parts of a scenario, we use transitions to go "back in time". Transitions are instantaneous intervals that can connect two states ragrdless of their chonological relasionship. 
```

Comment:
regardless of their chronological relationship.

Reply:
done in f0feab7